perfkitbenchmarker/container_service.py

```diff
@@ -31,6 +31,7 @@
 import logging
 import multiprocessing
 from multiprocessing import synchronize
+multiprocessing.set_start_method("fork", force=True)
```

Comment:
What happens if you *don't* do this?

Also, I see that https://docs.python.org/3/library/multiprocessing.html claims:

> The default start method will change away from fork in Python 3.14. Code that requires fork should explicitly specify that via get_context() or set_start_method().

I wonder if we should get this to work with `spawn` instead...